query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java

```diff
@@ -250,6 +257,7 @@ private String convertOperatorToString(Operator operator) {
         return "<=";
       case LIKE:
         return REGEX_OPERATOR;
+      case NOT_CONTAINS_KEY:
```

Comment:
These cases aren't right. It seems like they just got parked here so as not to throw since they're not used in the existing code and this is called prematurely. NOT_CONTAINS_KEY becomes `!=` and CONTAINS_KEY becomes `=` (the only difference being that we change the col translation to add on __keys). CONTAINS_KEYVALUE becomes multiple expressions, so could remain until removed.

Reply:
resolved